Classes/GHNSString+URL.h

```diff
@@ -53,9 +57,11 @@
  
  @see NSURL#gh_encode:
  */
-- (NSString *)gh_URLEncode;
+- (NSString *)gh_URLEncode DEPRECATED_MSG_ATTRIBUTE("Use -[NSString stringByAddingPercentEncodingWithAllowedCharacters:] with the appropriate NSCharacterSet for each part of the URL. You can't encode a whole URL properly.");
```

Comment:
It looks like we're barely using this, want to just delete it instead of deprecating it?